examples/floyd_warshall_df.impl.h

```diff
@@ -106,7 +108,22 @@ class Initiator : public Op<int,
     // making x_ready for all the blocks (for function calls A, B, C, and D)
     // This triggers for the immediate execution of function A at tile [0, 0]. But
     // functions B, C, and D have other dependencies to meet before execution; They wait
-    for (int i = 0; i < iterations; ++i) {
+
+    std::for_each(std::execution::par, adjacency_matrix_ttg->get().begin(), adjacency_matrix_ttg->get().end(), 
+      [&out](const std::pair<std::pair<int,int>, BlockMatrix<T>>& kv) {
+        auto [i,j] = kv.first;
+        if (i == 0 && j == 0) {  // A function call
+          ::send<0>(Key(std::make_pair(std::make_pair(i, j), 0)), kv.second, out);
+        } else if (i == 0) {  // B function call
+          ::send<1>(Key(std::make_pair(std::make_pair(i, j), 0)), kv.second, out);
+        } else if (j == 0) {  // C function call
+          ::send<2>(Key(std::make_pair(std::make_pair(i, j), 0)), kv.second, out);
+        } else {  // D function call
+          ::send<3>(Key(std::make_pair(std::make_pair(i, j), 0)), kv.second, out);
+        }
+    });
+      
+    /*for (int i = 0; i < iterations; ++i) {
```

Comment:
This block is commented out: if we don't need need, let's just remove it in the examples? Or do we want to keep it for a reason?